examples/plot_importing_data.py

```diff
@@ -13,17 +13,17 @@
 endtime = starttime + timedelta(hours=12)
 probe = '2'
 
-data = helios.corefit(probe, starttime, endtime)
+plot_data = helios.corefit(probe, starttime, endtime)
```

Comment:
Could this be `corefit` instead of `plot_data`? I think it will make the code look a bit more readable.